models/tags/Tags.ts

```diff
@@ -0,0 +1,4 @@
+export default interface Tags {
+    tag: string,
+    retags: number
```

Comment:
Can we change this to "frequency", instead of "retags"?

Reply:
Made the changes as per comments